src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -38,7 +38,7 @@ public void intakeSpin(double percentVoltage) {
 
   @Override
   public void initDefaultCommand() {
-    // Set the default command for a subsystem here.
+    setDefaultCommand(new IntakeCommand());
```

Comment:
don't set as default command

Reply:
why can't the intake command be set as a default command? the intake will not receive voltage unless a certain button is held, negating the need for using whileHeld in the OI

Reply:
this is a redundancy and you don't want to set to default command unless absolutely necessary. This means the command will be constantly run in every loop. If you can reduce the amount of stuff being iterated through, you effectively increase efficiency. Also whileHeld is just a better way to code it for this purpose.